services/nomad/build/buildbot.nomad

```diff
@@ -0,0 +1,164 @@
+        ports = ["http", "worker"]
+      }
+
+      volume_mount {
+        volume = "buildbot_database"
+        destination = "/db"
+      }
+
+      template {
+        data = <<EOF
+[buildbot]
+worker-port = {{ env "NOMAD_PORT_worker" }}
+www-port = {{ env "NOMAD_PORT_http" }}
+# TODO
+#url = http://build.voidlinux.org/
+
+[irc]
+host = irc.libera.chat
+# TODO
+# nick = void-builder
+# channel = #xbps
+# authz = duncaen maldridge
+nick = abby-testbot
+channel = ###abby-testbot
+authz = duncaen maldridge abby
+EOF
+        destination = "local/config.ini"
+      }
+
+      template {
+        data = <<EOF
+{
+  "workers": [
+    { "name": "glibc",   "max-builds": 4 },
+    { "name": "musl",    "max-builds": 3 },
+    { "name": "aarch64", "max-builds": 2 }
+  ],
+  "builders": [
+    { "name": "x86_64",       "host": "x86_64",                                "sync": false, "worker": "glibc"                         },
+    { "name": "i686",         "host": "i686",                                  "sync": false, "worker": "glibc"                         },
+    { "name": "x86_64-musl",  "host": "x86_64-musl",                           "sync": true,  "worker": "musl"                          },
+    { "name": "aarch64",      "host": "x86_64",      "target": "aarch64",      "sync": true,  "worker": "aarch64", "bootstrap_args": "" },
+    { "name": "aarch64-musl", "host": "x86_64-musl", "target": "aarch64-musl", "sync": true,  "worker": "aarch64", "bootstrap_args": "" }
+  ]
+}
+EOF
+/*
+    { "name": "x86_64",       "host": "x86_64",                                "sync": false, "worker": "glibc"                         },
+    { "name": "i686",         "host": "i686",                                  "sync": false, "worker": "glibc"                         },
+    { "name": "armv7l",       "host": "x86_64",      "target": "armv7l",       "sync": false, "worker": "glibc"                         },
+    { "name": "armv6l",       "host": "x86_64",      "target": "armv6l",       "sync": false, "worker": "glibc"                         },
+    { "name": "x86_64-musl",  "host": "x86_64-musl",                           "sync": true,  "worker": "musl"                          },
+    { "name": "armv7l-musl",  "host": "x86_64-musl", "target": "armv7l-musl",  "sync": true,  "worker": "musl"                          },
+    { "name": "armv6l-musl",  "host": "x86_64-musl", "target": "armv6l-musl",  "sync": true,  "worker": "musl"                          },
+    { "name": "aarch64",      "host": "x86_64",      "target": "aarch64",      "sync": true,  "worker": "aarch64", "bootstrap_args": "" },
+    { "name": "aarch64-musl", "host": "x86_64-musl", "target": "aarch64-musl", "sync": true,  "worker": "aarch64", "bootstrap_args": "" }
+*/
+        destination = "local/workers.json"
+      }
+
+      template {
+        data = file("master.cfg")
+        destination = "local/master.cfg"
+      }
+
+      template {
+        data = "buildbot master"
```

Comment:
Consider making this a more useful string that identifies that its Void's, running on a particular machine, and what version its running at.